docs/sign-up-and-onboard-to-techpass.md

```diff
@@ -28,7 +28,7 @@ Following are the prerequisites to sign up for TechPass and complete the onboard
 
 3. Indicate if you want to onboard your Internet Device to SEED and then select **I'm not a robot**.
 
-   !> To access services such as SGTS and GCC 2.0 resources through an Internet Device, you need to onboard that device to SEED.
+   !> - To access services such as SGTS and GCC 2.0 resources through an Internet Device, you need to onboard that device to SEED.<br> - If your email is not eligible for SEED (e.g. a schools.gov.sg), you will not see the option to onboard to SEED.
```

Comment:
If a @schools.gov.sg user really need SEED to access SGTS or GCC2.0 then we should indicate how the user should get a Member account.